.github/workflows/image-cleanup.yml

```diff
@@ -0,0 +1,28 @@
+name: Clean up PR-scoped test images
+
+on:
+  pull_request_target:
+    types:
+      - closed
+
+jobs:
+  delete-images:
+    name: Delete PR-scoped test images
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Delete Docker images
+        uses: r26d/ghcr-delete-image-action@v1.2.2
+        with:
+          owner: ${{ github.repository_owner }}
+          name: cryostat-operator
+          token: ${{ secrets.GHCR_PR_TOKEN }}
+          ignore-missing-package: true
+          tag-regex: pr-${{ github.event.number }}-.*
```

Comment:
Good idea! Please update the test image tags in `ci.yaml` to include this prefix. For eaxmple, the bundle image:

https://github.com/cryostatio/cryostat-operator/blob/2d386930dc96f0dcaf937987ec35874006c53b61/.github/workflows/ci.yaml#L120

Reply:
I would search for any tag with `ci-$GITHUB_SHA` and replace them with `pr-${{ github.event.number }}-$GITHUB_SHA`